### DISCUSSION TITLE
Entity documentation

### DISCUSSION THREAD

**Discussion by thomaskilian** [binsentsu/home-assistant-solaredge-modbus#230] [Q&A]
I just have installed the assistant and it shows me 200+ entities - phew! Quite some I can interpret, but by far not all of them. Can I find some doc somewhere? And also, is it possibile to hide most/many of them easily? I am loathe to click somwhere for 200+ times...

**Comment by dnamat88** [comment]
same here! i need to know the same value that i see in app solaredge so i can re-create the app in home assistant.

**Comment by thomaskilian** [comment]
to whom it may concern: [https://knowledge-center.solaredge.com/sites/kc/files/sunspec-implementation-technical-note.pdf](url)detalis a lot (I did not really read it). But it at least explains the Status (4=Inverter is ON and producing power) which is good to know...
